src/node.cc

```diff
@@ -4377,6 +4378,16 @@ int Start(int argc, char** argv) {
 
   CHECK_GT(argc, 0);
 
+#ifdef NODE_ENABLE_LARGE_CODE_PAGES
+  if (node::largepages::isLargePagesEnabled()) {
+    if ((node::largepages::map_static_code_to_large_pages()) != 0) {
+      fprintf(stderr, "Mapping of static code to large pages failed.\n");
+      fprintf(stderr, "Reverting to default page size\n");
+    }
+  }
+#endif
+
+
```

Comment:
any reason why we are doing it here, as opposed to the very beginning of everything, in main?

Reply:
No particular reason. This was a place where other things were initialized like VTune support.